vigetlabs/wordpress-site-starter
# [#1] Blueprint Build tools for UI

# Summary

This does not "style" any of the blocks but sets up the all the build tools to style our new blocks. 

- Fixes the Vite/ddev build so it works, but we should think about maybe moving to a package for adding Vite files to the PHP that will be in - https://github.com/vigetlabs/wordpress-site-starter/issues/2
- Set up to watch for changes in `theme-json` to rebuild `theme.json` change
- On `npm run build` we build the  `theme.json` file
- Updates how we pull in fonts to WP default (we could set these as var in TW?)
- Tab Width to 2
- Changed from mariadb to MySQL 8 - you may loose any data in your local DB unless you export and reimport it
- AlpineJS

## Issues

* #1 

## Testing Instructions

1. Change to the wp-starter theme.
2. Make sure ddev is restarted. 
3. Make edits to any of the files inside `theme-json` make sure changes get updated in `theme.json`.
4. Add [AplineJS code](https://alpinejs.dev/start-here#building-a-counter) to any WP frontend file and make sure it works. 

## Screenshots

NA
